src/version.rs

```diff
@@ -0,0 +1,14 @@
+// Copyright 2018 the Deno authors. All rights reserved. MIT license.
+use binding;
+use std::ffi::CStr;
+
+// This is the source of truth for the Deno version. Ignore the value in Cargo.toml.
+const DENO_VERSION: &str = "0.0.1";
```

Comment:
This is also fine for now - but for a really nice release pipeline it should live in an external file.
There are some rust macros that let you include an external file so we should probably go for that.